tasks/add-missing-file-checksums/package.json

```diff
@@ -41,6 +41,8 @@
   },
   "dependencies": {
     "@cumulus/aws-client": "3.0.0",
-    "@cumulus/cumulus-message-adapter-js": "1.3.1"
+    "@cumulus/cumulus-message-adapter-js": "1.3.2",
+    "@cumulus/types": "3.0.0",
+    "@types/aws-lambda": "^8.10.58"
```

Comment:
Should these two be dev dependencies? Also how were they not necessary before?

Reply:
Hmm, probably, yes.     

They weren't nessicary before, because the CMA-js package wasn't typed, so add-missing-file-checksums didn't need to know about some of them.